mcgill-en.csl

```diff
@@ -37,6 +37,18 @@
         <single>para</single>
         <multiple>paras</multiple>
       </term>
+      <term name="sub verbo" form="short">
+			  <single></single>
```

Comment:
yeah, that's what I suspected. Not happy with that. @rmzelle any thoughts on that?